web/gds-admin-ui/src/pages/app/details/BasicDetails/components/TrisaImplementationDetailsForm.js

```diff
@@ -0,0 +1,110 @@
+import { yupResolver } from '@hookform/resolvers/yup'
+import Field from 'components/Field'
+import { ModalCloseButton, ModalContext } from 'components/Modal'
+import { Alert, Button, Col, Form, Row } from 'react-bootstrap'
+import { useForm, useWatch } from 'react-hook-form'
+import { getTrisaImplementationDetailsInitialValue } from 'utils/form-references'
+import React from 'react'
+import * as yup from 'yup'
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import useSafeDispatch from 'hooks/useSafeDispatch'
+import { clearTrisaImplementationDetailsErrorMessage, updateTrisaImplementationDetailsResponse } from 'redux/vasp-details'
+import { getTrisaDetailsErrorState } from 'redux/selectors'
+
+const trisaEndpointPattern = /^(http[s]?:\/\/)?([a-zA-Z0-9.]+):((?!(0))[0-9]+)$/
```

Comment:
@elysee15 sorry for the confusion, I meant that the endpoint pattern should _not_ match any endpoint starting with "http://" or "https://" - this is a grpc endpoint, not an http endpoint, and one of the common errors is users putting https:// in the endpoint field mistakenly thinking that this is the same as a URL.